www/docs/mp/index.php

```diff
@@ -435,36 +435,6 @@
         MySociety\TheyWorkForYou\Renderer::output('mp/recent', $data);
         break;
 
-    case 'divisions':
```

Comment:
if we're dropping this then we should get rid of all the images in `www/docs/images/policies/` too as I think this is the only place that uses those.

Reply:
Yep - can clear those out. We don't have a plan to update the new ones.